### ISSUE TITLE
modify_Tfunc branch: make settings_cesm2.1+cocco.yaml (and .json)

### ISSUE THREAD

**Issue by mnlevy1981** [kristenkrumhardt/MARBL#3]
For POP, this will let users set `OCN_BGC_CONFIG = cesm2.1+cocco` to get the correct tunings out of the box. This is tied to two other updates:

1. Remove `QCaCO3_max_exp` (just set `QCaCO3_max` to 2.0 in the cocco settings file)
1. copy the current `settings_latest.yaml` file to `settings_cesm2.1.yaml`

**Comment by mnlevy1981** [comment]
This will also let us remove an if statement from `compute_autotroph_elemental_ratios()`:

```
if (autotroph_settings(auto_ind)%exp_calcifier) then
  QCaCO3(auto_ind,:) = min(auto_CaCO3(auto_ind,:) / (auto_C(auto_ind,:) + epsC), QCaCO3_max_exp)
else
  QCaCO3(auto_ind,:) = min(auto_CaCO3(auto_ind,:) / (auto_C(auto_ind,:) + epsC), QCaCO3_max)
end if
```

Will go back to being

```
QCaCO3(auto_ind,:) = min(auto_CaCO3(auto_ind,:) / (auto_C(auto_ind,:) + epsC), QCaCO3_max)
```

For all cases

**Comment by mnlevy1981** [comment]
Closed via #6
